docs/node-operators/get-started/setup-datanode.md

```diff
@@ -254,22 +318,22 @@ If you are running behind a firewall, you need to port forward 80+443 to the mac
 ::: 
 
 ## Data node retention profiles
-When starting a data node, you can choose the data retention configuration for your data node, depending on the use case for the node. The retention policy details can all be fine-tuned manually, as well.
+When initialising a data node, you can choose the data retention configuration for your data node, depending on the use case for the node. The retention policy details can all be fine-tuned manually, as well.
 
 There are 3 retention policy configurations:
-* **Standard (default)**: The node retains data according to the default retention policies, which assume a data node retains some data over time, but not all data
-* **Lite**: The node retains enough data to be able to provide the latest state to clients, and produce network history segments. This mode saves enough to provide the current state of accounts, assets, balances, delegations, liquidity provisions, live orders, margin levels, markets, network limits, network parameters, node details, parties, positions
-* **Archive**: The node retains all data
+* **Archive (default)**: The node retains all data and is the expected and only recommended retention profile for a public data node.
+* **Minimal**: The node retains only data about a network's current state. This can be useful for private data nodes who may want to only serve live data and stream changing states.
+* **Conservative**: The node does not retain all data and per-table rentention is set based on the expected table sizes. This can be useful for private data nodes who may want a customise their per-table data rentention based on their specific usecase.
```

Comment:
```suggestion
* **Archive (default)**: The node retains all data and is the expected, and only recommended, retention profile for a public data node.
* **Minimal**: The node retains only data about a network's current state. This can be useful for private data nodes that will serve live data and stream changing states.
* **Conservative**: The node does not retain all data and per-table rentention is set based on the expected table sizes. This can be useful for private data nodes that need customised per-table data rentention based on a specific usecase.
```